src/tag/Tag.js

```diff
@@ -1,9 +1,25 @@
 import PropTypes from 'prop-types';
 
-import { StyledTag, StyledTagLabel, StyledDeleteButton } from './Tag.styles';
+import {
+  StyledTag,
+  StyledTagLabel,
+  StyledDeleteButton,
+  StyledStartIcon,
+  StyledEndIcon
+} from './Tag.styles';
 
 const Tag = (props) => {
-  const { children, className, onClick, onDelete, variant, dataTestId } = props;
+  const {
+    children,
+    className,
+    onClick,
+    onDelete,
+    variant,
+    dataTestId,
+    startIcon,
+    endIcon,
+    size
```

Comment:
I talked with Andrea and she thinks this size isn't going to change ever so I kinda think we should just remove it. Thoughts?

Reply:
Yep no problemo.